README.md

```diff
@@ -309,6 +317,20 @@ https://medium.com/matic-network/understanding-dagger-453d90480c51
   - One way to prevent DDoS is to check state validation when user submits the transaction (before tx goes to pending pool)
   - Node must handle state check validation part while accepting transactions. If any node decides to skip this implementation, other nodes will ignore all future transactions from particular node.
 
+17. Exit
+
+  - Q. Why Matic doesn't have Exit despite of it is like Plasma?
+  - A. Exit is not necessarily needed for sidechain. It is one of security model to achieve customer asset preservation. In our model, Delegates reselection ceremony and fast chain resuming are the security model for assets.
```

Comment:
Matic will have exit queue but we will provide liquidity for small fees for those who want to exit instantly.

Reply:
We = Matic?

So, some of 10% revenue is to be pooled for instant "withdrawal(not exit)" subsidy?

Reply:
Lemme clarify,

```
1. We will ready for exit `priorityQueue`
2. Some operators will observe UTXOs and challenge to exit it
3. This exit module is not daily usage, if Matic chain is byzantine, this module is to be unlocked
4. Matic offer subsidy for `simple fast withdrawal` in that case
```

Is this interpretation correct?

Reply:
Yes.

Reply:
One way is to make sure you can burn old (before 2 checkpoint) balance only. This works best even in data-availability issue.